src/lib/intl-polyfill.js

```diff
@@ -0,0 +1,312 @@
+// this file should only be `required` in the format-time
+// when Intl.RelativeTimeFormat is not available (Safari < 14), but
+// we're not currently able to do the code splitting in www, and it
+// is always included. To reduce the amount of data that's loaded limit
+// the number of languages loaded to just the top few that are still using
```

Comment:
Nit - we should correct the comment now that we've added all the languages. Eventually this will end up in scratch-l10n, so it's not urgent.